polkadot/xcm/pallet-xcm/src/benchmarking.rs

```diff
@@ -257,163 +315,289 @@ benchmarks! {
-	}: _<RuntimeOrigin<T>>(execute_origin, Box::new(versioned_msg), Weight::MAX)
 
-	force_xcm_version {
-		let loc = T::reachable_dest().ok_or(
-			BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)),
-		)?;
+		#[extrinsic_call]
+		_(execute_origin as RuntimeOrigin<T>, Box::new(versioned_msg), Weight::MAX);
+
+		Ok(())
+	}
+
+	#[benchmark]
+	fn force_xcm_version() -> Result<(), BenchmarkError> {
+		let loc = T::reachable_dest()
+			.ok_or(BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)))?;
 		let xcm_version = 2;
-	}: _(RawOrigin::Root, Box::new(loc), xcm_version)
 
-	force_default_xcm_version {}: _(RawOrigin::Root, Some(2))
+		#[extrinsic_call]
+		_(RawOrigin::Root, Box::new(loc), xcm_version);
+
+		Ok(())
+	}
+
+	#[benchmark]
+	fn force_default_xcm_version() {
+		#[extrinsic_call]
+		_(RawOrigin::Root, Some(2))
+	}
 
-	force_subscribe_version_notify {
-		let versioned_loc: VersionedLocation = T::reachable_dest().ok_or(
-			BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)),
-		)?
-		.into();
+	#[benchmark]
+	fn force_subscribe_version_notify() -> Result<(), BenchmarkError> {
+		let versioned_loc: VersionedLocation = T::reachable_dest()
+			.ok_or(BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)))?
+			.into();
 
-		// Ensure that origin can send to destination (e.g. setup delivery fees, ensure router setup, ...)
+		// Ensure that origin can send to destination
+		// (e.g. setup delivery fees, ensure router setup, ...)
 		T::DeliveryHelper::ensure_successful_delivery(
 			&Default::default(),
 			&versioned_loc.clone().try_into().unwrap(),
 			FeeReason::ChargeFees,
 		);
 
-	}: _(RawOrigin::Root, Box::new(versioned_loc))
+		#[extrinsic_call]
+		_(RawOrigin::Root, Box::new(versioned_loc));
+
+		Ok(())
+	}
 
-	force_unsubscribe_version_notify {
-		let loc = T::reachable_dest().ok_or(
-			BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)),
-		)?;
+	#[benchmark]
+	fn force_unsubscribe_version_notify() -> Result<(), BenchmarkError> {
+		let loc = T::reachable_dest()
+			.ok_or(BenchmarkError::Override(BenchmarkResult::from_weight(Weight::MAX)))?;
 		let versioned_loc: VersionedLocation = loc.clone().into();
 
-		// Ensure that origin can send to destination (e.g. setup delivery fees, ensure router setup, ...)
+		// Ensure that origin can send to destination
+		// (e.g. setup delivery fees, ensure router setup, ...)
 		T::DeliveryHelper::ensure_successful_delivery(
 			&Default::default(),
 			&versioned_loc.clone().try_into().unwrap(),
 			FeeReason::ChargeFees,
 		);
 
 		let _ = crate::Pallet::<T>::request_version_notify(loc);
-	}: _(RawOrigin::Root, Box::new(versioned_loc))
 
-	force_suspension {}: _(RawOrigin::Root, true)
+		#[extrinsic_call]
+		_(RawOrigin::Root, Box::new(versioned_loc));
+
+		Ok(())
+	}
+
+	#[benchmark]
+	fn force_suspension() {
+		#[extrinsic_call]
+		_(RawOrigin::Root, true)
+	}
+
+	// todo check below more thoroughly
```

Comment:
you mean adding checks or checking yourself?
```suggestion

	// todo check below more thoroughly
```

Reply:
This is just a confusing diff. This line was there before my PR already, but I intended to not change anything besides the migration in this PR.

Reply:
I can't see it on master, it can have been remove after this PR was opened. I think we can just remove it.

Reply:
Yes, you are right! Thanks a lot! I have double-checked the code if any other comment might have sneaked in, but it looks fine.